rails6/fr/chapter09-optimization.adoc

```diff
@@ -201,12 +201,12 @@ class Api::V1::OrdersControllerTest < ActionDispatch::IntegrationTest
     get api_v1_orders_url, headers: { Authorization: JsonWebToken.encode(user_id: @order.user_id) }, as: :json
     assert_response :success
 
-    json_response = JSON.parse(response.body)
+    json_response = JSON.parse(response.body, symbolize_names: true)
     assert_equal @order.user.orders.count, json_response['data'].count
```

Comment:
I think now that we use the symbolize-names the `.count` does not work.

Should I remove the line, or do you prefer to do something else?

Reply:
With `assert_equal Product.count, json_response[:data].count` should be works